src/js/yeldaWebchatInjector.js

```diff
@@ -70,6 +71,12 @@ class YeldaChat {
 
     // Assistant Image Creation
     this.assistantImage = document.createElement('div')
+
+    // If the webchat should be opened directly, don't display the assistant image
+    if (data.shouldBeOpened) {
+      this.assistantImage.style.display = 'none'
+    }
+    
```

Comment:
If shouldBeOpened was set to true, the assistant image was displayed below the webchat instead of being hidden and only appear once the webchat is closed

Reply:
ok, then please add this explanation, it's useful !